src/layouts/partials/NotificationContent/NotificationContent.jsx

```diff
@@ -78,6 +78,7 @@ class NotificationContent extends PureComponent {
       case transaction instanceof TxExecModel:
         return new CurrentTransactionNotificationModel({
           id: transaction.hash(),
+          hash: transaction.hash(),
```

Comment:
You are already have 'id' with the same value. What was the reason to duplicate a field with different name?